actor-mini-profile.js

```diff
@@ -50,7 +51,7 @@ class ActorMiniProfile extends HTMLElement {
     // Actor icon
     const img = document.createElement('img')
     img.className = 'profile-mini-icon'
-    img.src = iconUrl
+    img.src = resolveP2PUrl(iconUrl)
```

Comment:
We shouldn't resolve to http unless there's an error loading. Maybe we should have a `p2p-image` custom element made which will attempt to render an `img` and fall back with the onerror handler? Then we can reuse the logic across pages.